spec/javascripts/NewService/formElements/BackendApiSelect.spec.jsx

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React from 'react'
+import {act} from 'react-dom/test-utils'
+import Enzyme, {mount, shallow, render} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {BackendApiSelect} from 'NewService/components/FormElements'
+import * as utils from 'utilities/utils'
```

Comment:
I don't think you need to import `utils`, the scope of this test is only the `select`, not the whole form.
And, if you import it, you should mock the implementation like [here](https://github.com/3scale/porta/blob/0dd700b542328e28d554aa13bb28d6c390637da8/spec/javascripts/NewService/NewServiceForm.spec.jsx#L12)